cert-data.yml

```diff
@@ -1,7 +1,7 @@
 ---
 edX/DemoX/Demo_Course:
   ISSUED_DATE: Jan. 1st, 1970
-  LONG_COURSE: Sample course
+  LONG_COURSE: Sample course with some utf-8 char like è
   LONG_ORG: Sample Org
 
 edX/DemoX_v3/Demo_Course_v3:
```

Comment:
Could you add a Unicode character to the v3 template's `LONG_COURSE`?

That way we can be sure that this introduces no regressions.